Welcome to
      ____              __
     / __/__  ___ _____/ /__
    _\ \/ _ \/ _ `/ __/  '_/
   /__ / .__/\_,_/_/ /_/\_\   version 2.1.1
      /_/

Using Python version 2.7.10 (default, Feb  6 2017 23:53:20)
SparkSession available as 'spark'.
>>> import tensorflow as tf
>>> from keras import optimizers
Using TensorFlow backend.
>>> from keras.models import load_model
>>> from keras.models import Sequential
>>> from keras.layers import Dense
>>> 
>>> data_rdd = spark.read.option("header",True).csv('/Users/sanket/Desktop/dlhackweek/image1.csv')
>>> train_rdd,test_rdd = data_rdd.randomSplit([0.9,0.1])
>>> train_rdd_x = train_rdd.select("0","1","2","3","4")
>>> train_rdd_y = train_rdd.select("Class")
>>> train_rdd_x_bc = sc.broadcast(train_rdd_x)
Traceback (most recent call last):
  File "/Users/sanket/home/fusion3.1/3.1.0-beta1/apps/spark-dist/python/pyspark/broadcast.py", line 81, in dump
    pickle.dump(value, f, 2)
  File "/Users/sanket/home/fusion3.1/3.1.0-beta1/apps/spark-dist/python/lib/py4j-0.10.4-src.zip/py4j/java_gateway.py", line 1133, in __call__
    answer, self.gateway_client, self.target_id, self.name)
  File "/Users/sanket/home/fusion3.1/3.1.0-beta1/apps/spark-dist/python/pyspark/sql/utils.py", line 63, in deco
    return f(*a, **kw)
  File "/Users/sanket/home/fusion3.1/3.1.0-beta1/apps/spark-dist/python/lib/py4j-0.10.4-src.zip/py4j/protocol.py", line 323, in get_return_value
    format(target_id, ".", name, value))
Py4JError: An error occurred while calling o44.__getnewargs__. Trace:
py4j.Py4JException: Method __getnewargs__([]) does not exist
	at py4j.reflection.ReflectionEngine.getMethod(ReflectionEngine.java:318)
	at py4j.reflection.ReflectionEngine.getMethod(ReflectionEngine.java:326)
	at py4j.Gateway.invoke(Gateway.java:272)
	at py4j.commands.AbstractCommand.invokeMethod(AbstractCommand.java:132)
	at py4j.commands.CallCommand.execute(CallCommand.java:79)
	at py4j.GatewayConnection.run(GatewayConnection.java:214)
	at java.lang.Thread.run(Thread.java:748)


Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/sanket/home/fusion3.1/3.1.0-beta1/apps/spark-dist/python/pyspark/context.py", line 775, in broadcast
    return Broadcast(self, value, self._pickled_broadcast_vars)
  File "/Users/sanket/home/fusion3.1/3.1.0-beta1/apps/spark-dist/python/pyspark/broadcast.py", line 72, in __init__
    self._path = self.dump(value, f)
  File "/Users/sanket/home/fusion3.1/3.1.0-beta1/apps/spark-dist/python/pyspark/broadcast.py", line 87, in dump
    raise pickle.PicklingError(msg)
cPickle.PicklingError: Could not serialize broadcast: Py4JError: An error occurred while calling o44.__getnewargs__. Trace:
py4j.Py4JException: Method __getnewargs__([]) does not exist
	at py4j.reflection.ReflectionEngine.getMethod(ReflectionEngine.java:318)
	at py4j.reflection.ReflectionEngine.getMethod(ReflectionEngine.java:326)
	at py4j.Gateway.invoke(Gateway.java:272)
	at py4j.commands.AbstractCommand.invokeMethod(AbstractCommand.java:132)
	at py4j.commands.CallCommand.execute(CallCommand.java:79)
	at py4j.GatewayConnection.run(GatewayConnection.java:214)
	at java.lang.Thread.run(Thread.java:748)


>>> 
